pkg/tracer/tracer.go

```diff
@@ -0,0 +1,35 @@
+package tracer
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/sdk/resource"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+)
+
+func TracerProvider(url, service string) (*tracesdk.TracerProvider, error) {
```

Comment:
Does OTEL provide a default TracerProvider based on the environment variables? https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/